views/events/performance_events.view.lkml

```diff
@@ -257,7 +257,7 @@ view_label: " Performance Events"
     label: "Number of API requests"
     description: "The number of api requests made by the client for an event."
     type: number
-    sql: ${TABLE}.num_of_request ;;
+    sql: COALESCE(${TABLE}.num_of_request, ${TABLE}.request_count) ;;
```

Comment:
where is `request_count` coming from?

Reply:
That's what I called the field for the `team_switch`/`channel_switch` events since I didn't realize you called it `num_of_request` for the `page_load` event. This measure will just use whichever of those values is defined when we get to Looker so that we won't have to deal with two fields that are similarly-named but populated in different cases